internal/convert_test.go

```diff
@@ -74,6 +74,43 @@ func TestGetDDL(t *testing.T) {
 	assert.ElementsMatch(t, normalize(e), normalize(ddl))
 }
 
+func TestGetFK(t *testing.T) {
+	conv := MakeConv()
+	conv.SpSchema["table1"] = ddl.CreateTable{
+		Name:     "table1",
+		ColNames: []string{"a", "b"},
+		ColDefs: map[string]ddl.ColumnDef{
+			"a": ddl.ColumnDef{Name: "a", T: ddl.Type{Name: ddl.Int64}},
+			"b": ddl.ColumnDef{Name: "b", T: ddl.Type{Name: ddl.Int64}},
+		},
```

Comment:
Consider dropping these extra fields from test struct since these are not relevant for the test at hand.

Reply:
Or did you intend to add a test with cyclic  foreign key dependencies so that table1 refers to table2 and vice versa?

I'm happy either way, but agree with Asheesh to trim the parts of the test that aren't relevant to what's being tested.

Reply:
merged foreign key test with GetDDL test as now we are using Tables flag